doc/conf.py

```diff
@@ -17,6 +17,9 @@
 import matplotlib
 import sphinx
 
+if sys.version_info <= (2, 7, 16):
```

Comment:
Would it be best to do `< (3, 0, 0)` to allow for any possible future python 2 releases?

Reply:
Yes you're right, let me update it.